editor/plugins/org.fusesource.ide.wsdl2rest/META-INF/MANIFEST.MF

```diff
@@ -11,40 +11,77 @@ Require-Bundle: org.fusesource.ide.foundation.core,
  org.eclipse.jdt,
  org.apache.xerces
 Bundle-ClassPath: .,
- lib/javaparser-core-2.5.1.jar,
- lib/commons-collections-3.2.2.jar,
- lib/stax2-api-3.1.4.jar,
- lib/commons-lang-2.4.jar,
- lib/velocity-1.7.jar,
- lib/wsdl4j-1.6.3.jar,
- lib/jaxb-xjc-2.2.11.jar,
- lib/xml-resolver-1.2.jar,
- lib/ant-nodeps-1.8.1.jar,
- lib/woodstox-core-5.0.3.jar,
- lib/args4j-2.33.jar,
- lib/camel-spring-2.21.0.fuse-000069.jar,
- lib/cxf-core-3.1.11.fuse-000226.jar,
- lib/cxf-codegen-plugin-3.1.11.fuse-000226.jar,
- lib/cxf-rt-databinding-jaxb-3.1.11.fuse-000226.jar,
- lib/cxf-rt-wsdl-3.1.11.fuse-000226.jar,
- lib/cxf-tools-common-3.1.11.fuse-000226.jar,
- lib/cxf-tools-wsdlto-core-3.1.11.fuse-000226.jar,
- lib/cxf-tools-wsdlto-frontend-jaxws-3.1.11.fuse-000226.jar,
- lib/javax.ws.rs-api-2.1.jar,
- lib/slf4j-api-1.7.25.jar,
- lib/woodstox-core-asl-4.4.1.jar,
- lib/jaxb-core-2.3.0.jar,
- lib/cxf-tools-wsdlto-databinding-jaxb-3.1.11.fuse-000226.jar,
- lib/xmlschema-core-2.2.1.jar,
- lib/cxf-tools-validator-3.1.11.fuse-000226.jar,
- lib/ant-1.10.1.jar,
- lib/ant-launcher-1.10.1.jar,
- lib/cxf-tools-wsdlto-frontend-javascript-3.1.11.fuse-000226.jar,
- lib/cxf-rt-bindings-soap-3.1.11.fuse-000226.jar,
- lib/wsdl2rest-impl.jar,
- lib/wsdl2rest-util.jar,
- lib/commons-validator-1.6.jar
+ libs/args4j.jar,
```

Comment:
why choosing to remove all the versions?
I see benefits that when upgrading versions, it will require less maintenance, I see downside that in the packaged plugin we don't have anymore an easy view of which version is embedded.
Other reasons?